### PR TITLE
global: add git attribute for easier file management

### PR DESCRIPTION
**Pull request by khancyr**

### CODE REVIEW

**Comment by Hwurzburg** [comment]
@peterbarker bump

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+text eol=lf
+
+# Declare files that will always have LF line endings on checkout.
+*.py text eol=lf
+*.sh text eol=lf
+*.bash text eol=lf
+
+*.bin -diff
+*.elf -diff
+*.hex -diff
```
Comment (peterbarker) [inline .gitattributes:11]: Are you expecting these in the Wiki?!

**Review by peterbarker — COMMENTED** [review]
I really don't like losing the history on that one file, but it's been painful in the past, and if it's required to pass these checks then OK.

File: .gitattributes (hunk 1 above)
Comment (khancyr) [reply .gitattributes]: right, who know... removed for now

**Pull request merged**